src/components/NoteExpand/NoteExpandContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+import { AppStateType } from '../../redux/redux-store';
+import ComeToHome from '../Common/ComeToHome/ComeToHome';
+import CommentsList from './CommentsBox/CommentsBox';
+import NoteContent from './NoteContent/NoteContent';
+import styles from './NoteExpandContainer.module.scss';
+
+const NoteExpandContainer = () => {
+  const { id } = useParams<{ id: string }>() || undefined;
+
+  const note = useSelector((state: AppStateType) => state.notesPage.notes.find((note) => id && note.id === +id));
+  return (
+    <div className={styles.wrapper}>
+      <ComeToHome />
+      {note !== undefined && id !== undefined ? (
```

Comment:
can be shorten to `{note && id &&  (<>
          <NoteContent name={note.name} content={note.content} id={id} />
          <CommentsList comments={note.comments} />
        </>
      )}`

Reply:
ready